tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/gifts/GiftInfo.kt

```diff
@@ -0,0 +1,72 @@
+package dev.inmo.tgbotapi.types.gifts
+
+import dev.inmo.tgbotapi.types.*
+import dev.inmo.tgbotapi.types.message.RawMessageEntities
+import dev.inmo.tgbotapi.types.message.textsources.TextSourcesList
+import dev.inmo.tgbotapi.types.message.toRawMessageEntities
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlin.jvm.JvmName
+
+
+@Serializable
+sealed interface GiftInfo {
+    val ownedGiftId: GiftId?
+    val gift: Gift
+
+    @Serializable
+    data class Regular(
+        @SerialName(giftField)
+        override val gift: Gift.Regular,
+        @SerialName(ownedGiftIdField)
+        override val ownedGiftId: GiftId? = null,
```

Comment:
This field will be presented only if the gift has been received and only for business accounts.

Currently, I do not understand how may we determine the receiver and the sender of gift in chats. Besides, I do not understand, is it possible to receive this event in any chat or in private only

In case I am correctly understand the logic of this API, currently realized `Regular` variant must be an `Surrogate`, `Regular` - sealed interface with two extenders: `ReceivedInBusinessAccount` and `Common`, first one with non-nullable `ownedGiftId`, second one without this field at all. For the first one must be created extender of `ChatEvent` named `BusinessAccountEvent`, for the second one must be used exists type as base one - `PrivateEvent` (in addition to `GiftInfo.Regular`